src/legacy.jl

```diff
@@ -0,0 +1,135 @@
+# This file is a part of Julia. License is MIT: https://julialang.org/license
+
+# StyledStrings provides a new way of handling colored output, however the
+# ~decade of colored printing in Julia before now has left us with a legacy,
+# and we should try to maintain compatibility as much as reasonable possible.
+
+module Legacy
+
+using ..StyledStrings: SimpleColor, Face, loadface!
+
+"""
+A mapping from 256-color codes to 8-bit colours, represented with
+`SimpleColor`-compatible `UInt32`s.
+"""
+const ANSI_256_COLORS = Dict{Int, UInt32}(
```

Comment:
Wouldn't this be better as a `UInt32` vector with 256 elements? When your dict keys are contiguous integers, what you actually want is an array.

Reply:
Ah yes, why didn't I think to do that initially? :laughing: